pkg/util/mount/mount.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/exec"
 )
 
-type Interface interface {
+type Mounter interface {
```

Comment:
The problem with this name is when it's used for composition. In our case we would have:

```
type System interface {
  mount.Interface
  util.Writer
}
```
If I'm interested in mounter interface I'd have to write ```system.Interface```. This suggests that it's and interface of some system, not of mounter, because we've lost package name here. With ```mount.Mounter``` we'd write ```system.Mounter``` which is far more readable IMO.